packages/shared/src/components/pagination/PaginationActions.tsx

```diff
@@ -35,15 +35,49 @@ export const PaginationActions = ({
           icon={<ArrowIcon className="-rotate-90" />}
           className="btn-tertiary"
           disabled={current === 1}
-          onClick={onNext}
+          onClick={onPrevious}
         />
         <Button
           {...buttonProps}
           icon={<ArrowIcon className="rotate-90" />}
           className="btn-tertiary"
           disabled={max === current}
+          onClick={onNext}
+        />
+      </div>
+    </div>
+  );
+};
+
+export interface InfinitePaginationActionsProps
```

Comment:
I had to create this one since our cursor paginator turns out does not support `total` count (even though the actual param is mentioned). Which makes sense. So the nuisance is we can not show `1/X` labels in the widget. I don't think it should be a problem generally? 

Alternative is to implement new paginator on the API but I think it would be a waste. 

<img width="303" alt="image" src="https://github.com/dailydotdev/apps/assets/9803078/032977b8-9420-44e0-a1e2-54a3e027f534">